components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -173,26 +164,9 @@ void BraveWalletService::SaveToPrefs(
       base64_cipher_seed);
 }
 
-void BraveWalletService::ResetCryptoWallets(
-    const base::FilePath& profile_path) {
-  const base::FilePath wallet_data_path = profile_path
-      .AppendASCII("Local Extension Settings")
-      .AppendASCII(ethereum_remote_client_extension_id);
-
-  base::PostTaskAndReplyWithResult(file_task_runner_.get(), FROM_HERE,
-      base::BindOnce(&ResetCryptoWalletsOnFileTaskRunner, wallet_data_path),
-      base::BindOnce(&BraveWalletService::OnCryptoWalletsReset,
-                     weak_factory_.GetWeakPtr()));
-}
-
-void BraveWalletService::OnCryptoWalletsReset(bool success) {
-  if (success) {
-    CloseTabsAndRestart();
-  }
-}
-
-void BraveWalletService::CloseTabsAndRestart() {
-  brave_wallet_delegate_->CloseTabsAndRestart();
+void BraveWalletService::ResetCryptoWallets() {
+  extensions::ExtensionPrefs::Get(context_)->DeleteExtensionPrefs(
+      ethereum_remote_client_extension_id);
```

Comment:
This wasn't introduced by you but could you add this dep?       `"//extensions/browser"` in `brave/components/brave_wallet/browser/BUILD.gn`?

Reply:
done 👍